requirements.txt

```diff
@@ -6,7 +6,7 @@ idna==3.7
 Jinja2==3.1.5
 MarkupSafe==2.0.1
 psycopg2-binary==2.9.3
-requests==2.31.0
+requests==2.32.2
```

Comment:
Updating to a newer requests fails the build, so I am toggling this PR into draft mode.

```python
 requests.exceptions.InvalidURL: Not supported URL scheme http+docker
```